src/renderer/library/components/settings/ReadingOptions/TextContent.tsx

```diff
@@ -0,0 +1,130 @@
+import fontList, { FONT_ID_DEFAULT, FONT_ID_VOID } from "readium-desktop/utils/fontList";
+import * as stylesReader from "readium-desktop/renderer/assets/styles/reader-app.css";
+
+const textContent= () => {
+    const {__, readerConfig} = this.props;
```

Comment:
- The first letter of the component must be in the upperCase.
- You cannot used `this.props` from a react functional component, `props` must be passed from function parameter and not from function binding `this` this keyword is used in class components instead.